cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -247,6 +248,14 @@ func GetEtcdPodSpec(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.A
 				staticpodutil.NewVolumeMount(etcdVolumeName, cfg.Etcd.Local.DataDir, false),
 				staticpodutil.NewVolumeMount(certsVolumeName, cfg.CertificatesDir+"/etcd", false),
 			},
+			Resources: v1.ResourceRequirements{
+				Requests: v1.ResourceList{
+					// TODO: make these values configurable
```

Comment:
please remove this TODO for now.
users can use the --experimental-patches functionality to apply patches during init/join/upgrade.
the feature can one day be in the kubeadm config too.